docs/graphcache/local-directives.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Local Directives
+order: 2
+---
```

Comment:
Should this be merged with local resolvers as the signature is almost identical?